server/include/array.h

```diff
@@ -64,7 +64,7 @@ extern "C" {
      * @param array the array to get the element from
      * @param index the index to get the element from
      */
```

Comment:
You may need to add a @return here to explain why the function is not a void function

Reply:
Not resolved